yb-voyager/cmd/endMigrationCommand.go

```diff
@@ -172,58 +204,155 @@ func backupLogFilesFn() {
+			continue
+		}
+
+		logFilePath := filepath.Join(exportDir, "logs", file.Name())
+		backupFilePath := filepath.Join(backupDir, "logs", file.Name())
+		err = os.Rename(logFilePath, backupFilePath)
+		if err != nil {
+			utils.ErrExit("end migration: moving log files: %v", err)
+		}
+	}
+}
+
+func askPassword(destination string, user string, envVar string) (string, error) {
+	if os.Getenv(envVar) != "" {
+		return os.Getenv(envVar), nil
+	}
+
+	if user == "" {
+		fmt.Printf("Password to connect to %s (In addition, you can also set the password using the environment variable '%s'): ",
+			destination, envVar)
+	} else {
+		fmt.Printf("Password to connect to '%s' user of %s (In addition, you can also set the password using the environment variable '%s'): ",
+			user, destination, envVar)
+	}
+	bytePassword, err := term.ReadPassword(int(syscall.Stdin))
+	if err != nil {
+		return "", fmt.Errorf("reading password: %w", err)
+	}
+	fmt.Print("\n")
+	return string(bytePassword), nil
 }
 
 func cleanupSourceDB(msr *metadb.MigrationStatusRecord) {
-	utils.PrintAndLog("cleaning up source db...")
+	utils.PrintAndLog("cleaning up voyager state from source db...")
 	source := msr.SourceDBConf
 	if source == nil {
 		log.Info("source db conf is not set. skipping cleanup")
 		return
 	}
-	err := source.DB().Connect()
+
+	var err error
+	source.Password = sourceDBPassword
+	if sourceDBPassword == "" {
+		source.Password, err = askPassword("source DB", source.User, "SOURCE_DB_PASSWORD")
+		if err != nil {
+			utils.ErrExit("end migration: getting source db password: %v", err)
+		}
+	}
+	err = source.DB().Connect()
 	if err != nil {
 		utils.ErrExit("end migration: connecting to source db: %v", err)
 	}
 	defer source.DB().Disconnect()
 	err = source.DB().ClearMigrationState(migrationUUID, exportDir)
 	if err != nil {
-		utils.PrintAndLog("end migration: clearing migration state from source db: %v", err)
-	}
-
-	if msr.YBCDCStreamID == "" {
-		log.Info("yugabytedb cdc stream id is not set. skipping deleting stream id")
-		return
-	}
-	ybCDCClient := dbzm.NewYugabyteDBCDCClient(exportDir, "", source.SSLRootCert, source.DBName, strings.Split(source.TableList, ",")[0], metaDB)
-	// TODO: check the error once streamID is expirted and ignore it
-	err = ybCDCClient.DeleteStreamID()
-	if err != nil {
-		utils.ErrExit("end migration: deleting yugabytedb cdc stream id: %v", err)
+		utils.ErrExit("end migration: clearing migration state from source db: %v", err)
 	}
 }
 
 func cleanupTargetDB(msr *metadb.MigrationStatusRecord) {
-	utils.PrintAndLog("cleaning up target db...")
+	utils.PrintAndLog("cleaning up voyager state from target db...")
 	if msr.TargetDBConf == nil {
 		log.Info("target db conf is not set. skipping cleanup")
 		return
 	}
-	tdb := tgtdb.NewTargetDB(msr.TargetDBConf)
-	err := tdb.Init()
+
+	var err error
+	tconf := msr.TargetDBConf
+	tconf.Password = targetDBPassword
+	if targetDBPassword == "" {
```

Comment:
Will it still be possible to support empty passwords?

Reply:
the user needs to provide something. `""` is also acceptable for empty passwords